_sources/CSPPracticeQuestions/exam24_2.rst

```diff
@@ -112,9 +112,9 @@ You will not be able to change your answers after you hit the "Finish Exam" butt
        :answer_d: The run-time first increases and then decreases as the input size is increased.
        :correct: a
        :feedback_a: Correct. This is true because every input requires the same amount of time.
-       :feedback_b: Incorrect. This is incorrect because time per step doesn't decrease with more steps.
+       :feedback_b: Incorrect. This is incorrect because time per step doesn't decrease with more inputs.
        :feedback_c: Incorrect. The number of steps an algorithm goes through does influence the the time an algorithm takes to run. 
-       :feedback_d: Incorrect. The number of steps increasing always increases the runtime of a algortihm in this case. 
+       :feedback_d: Incorrect. The number of steps increasing always increases the runtime of an algorithm in this case. 
```

Comment:
This feedback needs to be clearer --- needs to make it clear that we are talking about *this* algorithm that runs in linear time.